app/src/components/Footer.tsx

```diff
@@ -5,12 +5,15 @@ import {
 } from "../selectors/userInfoSelectors";
 import { hasAccess } from "../utils/utils";
 import { useAppSelector } from "../store";
+import { Link, useNavigate } from "react-router-dom";
```

Comment:
`eslint` complains:
```
'useNavigate' is defined but never used
```
Can this go? Do we really need this?

```suggestion
import { Link } from "react-router-dom";
```